### PR TITLE
Sara/quickfix/close jira

### PR DESCRIPTION
**Pull request by s-santillan**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR

Changing status from private beta to closed beta.

### CODE REVIEW

**Review by jsmoxon — APPROVED** [review]
Looks good to me!

File: docs/semgrep-cloud-platform/asana.md [hunk 1]
```diff
@@ -19,17 +19,17 @@ Object.entries(frontMatter).filter(
...
+# Create Asana tickets
 
-:::tip Try the Asana integration (beta)
-Existing customers can try the Asana integration for free. To enable this feature:
-1. Fill out the following form: [Request access to the Semgrep Asana integration private beta](https://get.semgrep.dev/Jira-asana-linear-private-beta.html).
-2. Contact your Technical Account Manager or your Account Executive and let them know you'd like to try the Asana integration out.
+:::tip Private beta
+This feature is in a closed beta. To request access:
+1. Fill out the following form: [Request access to the Semgrep Asana integration closed beta](https://get.semgrep.dev/Jira-asana-linear-private-beta.html).
+2. Contact your Technical Account Manager or your Account Executive and let them know you'd like to try the Jira integration out.
```
Comment (khorne3) [inline docs/semgrep-cloud-platform/asana.md:27]:
```suggestion
2. Contact your Technical Account Manager or your Account Executive and let them know you'd like to try the Asana integration out.
```

File: docs/semgrep-cloud-platform/linear.md [hunk 2]
```diff
@@ -19,17 +19,17 @@ Object.entries(frontMatter).filter(
...
+# Create Linear tickets
 
-:::tip Try the Linear integration (private beta)
-Existing customers can try the Linear integration for free. To enable this feature:
-1. Fill out the following form: [Request access to the Semgrep Linear integration private beta](https://get.semgrep.dev/Jira-asana-linear-private-beta.html).
-2. Contact your Technical Account Manager or your Account Executive and let them know you'd like to try the Linear integration out.
+:::tip Private beta
+This feature is in a closed beta. To request access:
+1. Fill out the following form: [Request access to the Semgrep Linear integration closed beta](https://get.semgrep.dev/Jira-asana-linear-private-beta.html).
+2. Contact your Technical Account Manager or your Account Executive and let them know you'd like to try the Jira integration out.
```
Comment (khorne3) [inline docs/semgrep-cloud-platform/linear.md:27]:
```suggestion
2. Contact your Technical Account Manager or your Account Executive and let them know you'd like to try the Linear integration out.
```

File: docs/semgrep-cloud-platform/ticketing.md [hunk 3]
```diff
@@ -31,3 +31,7 @@ Semgrep Cloud Platform can create tickets in the following systems:
 * [Asana](/semgrep-cloud-platform/asana)
 * [Jira](/semgrep-cloud-platform/jira)
 * [Linear](/semgrep-cloud-platform/linear)
+
+These features are in a closed beta. To request access:
+1. Fill out the following form: [Request access to the Semgrep ticketing integrations closed beta](https://get.semgrep.dev/Jira-asana-linear-private-beta.html).
+2. Contact your Technical Account Manager or your Account Executive and let them know you'd like to try the Jira integration out.
```
Comment (khorne3) [inline docs/semgrep-cloud-platform/ticketing.md:37]:
```suggestion
2. Contact your Technical Account Manager or your Account Executive and let them know you'd like to try the integration out.
```

File: docs/semgrep-code/java.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
```
Comment (khorne3) [inline docs/semgrep-code/java.md:1]: Are these Java docs supposed to be here?

Comment (s-santillan) [reply docs/semgrep-code/java.md:1]: cry

Comment (s-santillan) [reply docs/semgrep-code/java.md:1]: i branched out of the wrong place again

**Pull request merged**